src/constants/triggers.ts

```diff
@@ -0,0 +1,6 @@
+export enum Triggers {
+  COLLECTION_VIEW = 'collection-view',
+  ITEM_VIEW = 'item-view',
+  LINK_OPEN = 'link-open',
+  ITEM_DOWNLOAD = 'item-download',
```

Comment:
We try to apply the PascalCase Naming convention for enums. 
```suggestion
  CollectionView = 'collection-view',
  ItemView = 'item-view',
  LinkOpen = 'link-open',
  ItemDownload = 'item-download',
```